Source/ReferenceConnectorService/Startup.cs

```diff
@@ -23,18 +18,22 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMvc();

```

Comment:
question: Out of interest, why change this one? .AddMvc() isn't deprecated, is it?

Reply:
AddMvc is still valid and a change wasn't strictly necessary.  AddMvc adds middleware support for a complete MVC app, including views, Razor pages, etc.  AddControllers() scales this back and adds just enough support for simple API apps.